tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -155,12 +155,14 @@ def test_poly1d_get2(self, dtype):
 
     @testing.for_all_dtypes()
     def test_poly1d_get3(self, dtype):
-        arr = testing.shaped_arange((5,), cupy, dtype)
+        a1 = testing.shaped_arange((5,), cupy, dtype)
+        b1 = cupy.poly1d(a1, variable='z')
         out = numpy.poly1d(numpy.empty((5,), dtype))
-        cupy.poly1d(arr, variable='z').get(out=out)
-        a = testing.shaped_arange((5,), numpy, dtype)
-        b = numpy.poly1d(a, variable='z')
-        assert out == b
```

Comment:
Could you explain why the CI failed in the previous version?

Reply:
I think because of the missing `else`. When I printed out before the assert statement. I found that it is printed twice for each dtype, the second print was exactly the same as the wrong test result of Jenkins.
As for https://github.com/cupy/cupy/pull/3466/commits/55e82569b01f4c4dcc10a772cb0b075531861442, I don't think it has an effect.
Not sure though because tests pass locally in both cases.

Reply:
If you think https://github.com/cupy/cupy/pull/3466/commits/72dc3e77eecb05feb092b6b5e3f321ae02acc650 fixed the issue, could you revert https://github.com/cupy/cupy/pull/3466/commits/55e82569b01f4c4dcc10a772cb0b075531861442?

Reply:
If you think there is something with the test or `get()` itself, please point it out.

Reply:
The property method `coeffs` in the newest NumPy stable version (used in Jenkins test `cupy-py3`) returns `self._coeffs` itself:
https://github.com/numpy/numpy/blob/c54a1acb713497f8c763574a4ba35fdfc864e53f/numpy/lib/polynomial.py#L1110-L1113
However, that in NumPy 1.15.4 (used in `cupy-py35`) returns a copy, so `out.coeffs` is not updated.
https://github.com/numpy/numpy/blob/3430d78c01a3b9a19adad75f1acb5ae18286da73/numpy/lib/polynomial.py#L1036-L1039

Reply:
How about dropping `out` keyword argument of `get` method?